# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iproute2/install-tc.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iproute2-2.6.34/tc/Makefile.vanilla	2010-07-17 16:19:32.160160079 +0000
+++ iproute2-2.6.34/tc/Makefile	2010-07-17 16:20:08.312175125 +0000
@@ -99,6 +99,7 @@
 	$(AR) rcs $@ $(TCLIB)
 
 install: all
+	install -m 0755 tc $(DESTDIR)$(SBINDIR)
 	echo mkdir -p $(MODDESTDIR)
 	echo install -m 0755 tc $(DESTDIR)$(SBINDIR)
 	for i in $(TCSO); \
